spec/lib/failover_databases_spec.rb

```diff
@@ -0,0 +1,132 @@
+describe FailoverDatabases do
+  let(:connection) { ActiveRecord::Base.connection }
+
+  before(:all) do
```

Comment:
I am trying to avoid creating new schema, table and inserting test record for each test.
Since ```before(:all)``` is not in transaction, I do manual clean-up on ```after(:all)```  - drop schema and return connection to connection pull.